doc/usermanual/lighttable/panels/tagging.xml

```diff
@@ -28,16 +28,17 @@
           <row>
             <entry>
               <para>
-                This panel is for managing tags on images. Tags are stored in both sidecar files
-                (.xmp) and within the darktable database for a faster access.
+                This panel allows managing tags on images. Tags are stored in both XMP sidecar files
+                and within the darktable database.
               </para>
               <para>
-                The goal of tags is to add information to the images within a given keyword set.
-                But when the number of tags becomes large it is interesting to organize them in a hierachical manner, as a tree.
-                The parent tags may not add information to the image, for example the tag <quote>color</quote>
-                in <quote>color|white</quote>. In this case <quote>color</quote> will automatically
-                seen as <quote>category</quote> if it is not in the dictionary itself. If exist as a tag, it can be set
-                explicitly as a <quote>category</quote> (see create or edit command).
+                Tags provide the means of adding information to the images within a given keyword dictionary.
+                You can manage tags as a hierarchical tree, which is useful when their number becomes large.
+              </para>
+              <para>
+                Unlike child tags, parent tags do not add information to the image, but rather represent categories.
+                Consider the <quote>color|white</quote> tag where <quote>color</quote> surves as a category 
```

Comment:
Suggestion: For instance, in the <quote>color|white</quote> tag, <quote>color</quote> is a category for the <quote>white</quote> tag.
In any case: I think you mean "serve" and not "surve"?

Reply:
merged